### PR TITLE
Give units to total filesize

### PR DESCRIPTION
**Pull request by triazo**

### CODE REVIEW

File: template.go [hunk 1]
```diff
@@ -35,6 +37,18 @@ func siacoinString(siacoins types.Currency) string {
 	return fmt.Sprintf("%f %s", coins, coinPostfixes[i])
 }
 
+func byteString(numBytes uint64) string {
```
Comment (DavidVorick) [inline template.go:40]: bit of a confusing name, but I'm not sure what would be better.

Comment (triazo) [inline template.go:40]: also fitting this pattern are rateString and siacoinString, which are very similar. Maybe byteNumString?

Comment (DavidVorick) [inline template.go:40]: that's worse :p

we'll keep it this way for now

Comment (lukechampine) [inline template.go:40]: in siac it is called `filesizeBytes` and in Sia-UI it is called `formatBytes`.

**Pull request merged**